release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -477,6 +477,40 @@ Available as a Technology Preview, the Insights Runtime Extractor feature gather
 Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers.
 For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[_Enabling features using feature gates_].
 
+[id="ocp-release-notes-insights-operator-rapid-recommendations_{context}"]
+==== Rapid Recommendations
+
+In this release, enhancements have been made to the Rapid Recommendations mechanism for remotely configuring the rules that determine the data that the Insights Operator collects.
+
+The Rapid Recommendations feature is version-independent, and builds on the existing conditional data gathering mechanism. 
+The Insights Operator connects to a secure remote endpoint service running on /console.redhat.com to retrieve definitions that contain the rules for determining which container log messages are filtered and collected by Red{nbsp}Hat.
+
+The conditional data-gathering definitions get configured through an attribute named `conditionalGathererEndpoint` in the link:https://github.com/openshift/insights-operator/blob/master/config/pod.yaml[`pod.yml`] configuration file.
+
+[source,bash]
+----
+conditionalGathererEndpoint: https://console.redhat.com/api/gathering/v2/%s/gathering_rules
+----
+
+[NOTE]
+====
+In earlier iterations, the rules for determining the data that the Insights Operator collects were hard-coded and tied to the corresponding {product-title} version.
+====
+
+The preconfigured endpoint URL now provides a placeholder (`%s`) for defining a target version of {product-title}.
+
+[id="ocp-release-notes-insights-operator-more_data_collected_and_recommendations_added_{context}"]
+==== More data collected and recommendations added
```

Comment:
This is a pretty vague and general heading. I'd be more specific or really you could collapse this into the enhancements piece above.

Reply:
This heading is consistent with previous release notes and in other places for Insights prod docs:

https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html#more-data-collected-and-recommendations-added

We have a large epic coming for how we document the data that Insights collects in 4.19. Will add this feedback to that epic. Thanks.